src/pyetm/models/scenario_packer.py

```diff
@@ -86,9 +89,9 @@ def _to_dataframe(self, values='') -> pd.DataFrame:
 
 
 class OutputCurvesPack(Packable):
-    key: ClassVar[str] = 'output_curves'
+    key: ClassVar[str] = "output_curves"
 
-    def _to_dataframe(self, values='') -> pd.DataFrame:
+    def _to_dataframe(self, values="") -> pd.DataFrame:
         """PACKS ONLY FIRST SCENARIO"""
         for scenario in self.scenarios:
             series_list = list(scenario.carrier_curves_series())
```

Comment:
```suggestion
            series_list = list(scenario.all_output_curves())
```
Let's change it to this for now. Later we make sure that users can specify which they want exactly.